### PR TITLE
Introduction of decimal round

### PR DESCRIPTION
**Pull request by Corniel**
By default, .NET support rounding of floating points (including `decimal`s).
However, for some domains this support is too limited. To overcome this, Qowaiv
has the static `DecimalRound` helper class, containing extension methods for rounding.

#### ‘Negative’ decimals
To round tenfold, hundredfold, etc. precision, a negative amount of decimals
can be specified:
``` C#
var tenfold = 1245.346m.Round(-1); // 1250m
var hundredfold = 1209m.Round(-2); // 1200m
```

#### Multiple of
Rounding to a multiple of is supported:
``` C#
var multipleOf = 123.5m.RoundToMultiple(5m); //   125.0m
var multiple25 = 123.5m.RoundToMultiple(2.5m); // 122.5m
```

Note that this is also required by #96 . I think it is quite some functionality by its own, and deserves a separate pull request.

### CODE REVIEW

**Comment by Corniel** [comment]
Biggest concern to me is the potential confusing difference between:

```C#
var rounded = 13.456m.Round(2); // 13.46m;
var multiple = 13.456m.Round(2m); // 12m;
```

You can opt for a longer method name like `RoundToMultpleOf()`, but I'm not sure about that either.

**Comment by Sjaaky** [comment]
My vote goes to `RoundToMultiple()`.

File: README.md [hunk 1]
```diff
@@ -210,6 +210,38 @@ also support the use of SQL wildcard characters _ and %.
...
+```
+
+#### Extra rounding methods
+.NET supports rounding to even (Bankers rounding) and away from zero out-of-the-box.
+Rounding methods like ceiling, floor, and truncate have limited support (0 decimals only),
+and a lot others (to odd, half-way up, half-way down, e.o.) are missing at all.
+By specifying the `DecimalRounding` 13 ways are supported.
+
+``` c#
+var toOdd = 23.0455m.Round(3, DecimalRouding.ToOdd); // 23.045m
```
Comment (LauraXiulan) [inline README.md]: DecimalRounding

File: README.md [hunk 2]
```diff
@@ -210,6 +210,38 @@ also support the use of SQL wildcard characters _ and %.
...
+Rounding to a multiple of is supported:
+``` C#
+var multipleOf = 123.5m.RoundToMultiple(5m); //   125.0m
+var multiple25 = 123.5m.RoundToMultiple(2.5m); // 122.5m
+```
+
+#### Extra rounding methods
+.NET supports rounding to even (Bankers rounding) and away from zero out-of-the-box.
+Rounding methods like ceiling, floor, and truncate have limited support (0 decimals only),
+and a lot others (to odd, half-way up, half-way down, e.o.) are missing at all.
```
Comment (LauraXiulan) [inline README.md]: 'and many others .... are missing.'

**Pull request merged**